cni/network/network.go

```diff
@@ -685,9 +685,11 @@ func (plugin *NetPlugin) findMasterInterface(opt *createEpInfoOpt) string {
 		return plugin.findMasterInterfaceBySubnet(opt.ipamAddConfig.nwCfg, &opt.ifInfo.HostSubnetPrefix)
 	case cns.NodeNetworkInterfaceFrontendNIC, cns.NodeNetworkInterfaceAccelnetFrontendNIC:
 		return plugin.findInterfaceByMAC(opt.ifInfo.MacAddress.String())
-	case cns.BackendNIC: // TODO: how to find interface with IB NIC by mac address
-		opt.ifInfo.Name = ibInterfacePrefix + strconv.Itoa(opt.endpointIndex)
-		return opt.ifInfo.Name
+	case cns.BackendNIC:
+		// if windows swiftv2 has right network drivers, there will be an NDIS interface while the VFs are mounted
+		// when the VF is dismounted, this interface will go away
+		// return an unique interface name to containerd
+		return ibInterfacePrefix + strconv.Itoa(opt.endpointIndex)
```

Comment:
will it break if we didnt set `opt.ifinfo.Name`

Reply:
it will not break because:
findMasterInterface() will return masterIfName and masterIfName is directly used in endpointInfo struct:
endpointInfo := network.EndpointInfo{
		NetworkID:                     opt.networkID,
		Mode:                          opt.ipamAddConfig.nwCfg.Mode,
		MasterIfName:                  **masterIfName,**  => this interface is returned from findMasterInterface
so we do not need to set opt.ifinfo.Name

Reply:
no.. i think ifInfo is a pointer and passed to different places

Reply:
`ifInfo.Name` not passed to endpointinfo. so should be good.

Reply:
I believe opt.ifinfo.Name isn't used in _windows_ -- Currently, ibnic (where we perform this change) is only used in windows (Paul can confirm). After a reference search on windows for interface name, I only see usages for ifInfo name in endpoint.go's pretty string function in windows. In linux ifName is used in secondary endpoint client. Even on linux, it seems like the interface name field is just being set after getting it based on a system call, rather than from epInfo/opt.ifinfo (and then is never used).